src/sonic-yang-mgmt/sonic-cfg-help

```diff
@@ -54,7 +54,14 @@ class SonicCfgDescriber:
                 self.print_field_desc(table['list'], field)
                 print()
         elif table.get('container') is not None:
-            self.print_field_desc(table.get('container'), field)
+            if isinstance(table['container'], dict):
+                print("key - " + table['container'].get('@name'))
+                self.print_field_desc(table.get('container'), field)
+            elif isinstance(table['container'], list):
+                for c in table['container']:
+                    print("key - " + c.get('@name'))
```

Comment:
Can we filter the common logic b/w these two, i.e. handling a single container dictinto a separate function and re-use in both the cases?

Reply:
print_field_desc is purpose is to do that - print a container. In the list case we loop through the list and call this API. The only additional line is printing the key. I prefer to keep this outside of print_field_desc function and so you had one line of duplication.

Reply:
I see, okay